app/controllers/authenticate_controller.rb

```diff
@@ -19,6 +18,12 @@ def index
     render json: authenticators
   end
 
+  def login
+    result = perform_basic_authn
+    raise Unauthorized, "Client not authenticated" unless authentication.authenticated?
+    render text: result.authentication_key
+  end
+
   def authenticate
     authentication_token = ::Authentication::Strategy.new(
```

Comment:
The creation of `Authentication::Strategy` can be moved to a private method an memoized.  It could even be put in an initializer but I'm a bit hazy on the best practice around doing that.